rpcs3/rpcs3qt/trophy_manager_dialog.cpp

```diff
@@ -321,7 +321,16 @@ trophy_manager_dialog::trophy_manager_dialog(std::shared_ptr<gui_settings> gui_s
 		m_game_combo->setCurrentText(m_game_table->item(m_game_table->selectedItems().first()->row(), GameColumns::GameName)->text());
 	});
 
-	RepaintUI(true);
+	RepaintUI();
+
+	// Fill trophy list
+	auto loadProgress = new QProgressDialog(
+			"Loading trophy data, please wait...", QString(), 0, 1, this,
```

Comment:
Text that is visible to the user must be used like this tr("your text") in order to make the application futureproof for localizations. Excluded are log messages.